calyx-py/test/correctness/queues/binheap/pifo.py

```diff
@@ -98,55 +99,44 @@ def insert_binheap_pifo(prog, name, boundary=BOUNDARY, queue_size_factor=FACTOR)
     r_b_incr_2 = comp.incr(r_b, 2)
 
     turn = comp.reg(1, "turn")
-    turn_neq_flow_out = comp.neq_use(turn.out, flow_out.out)
+    turn_eq_flow_out = comp.eq_use(turn.out, flow_out.out)
```

Comment:
I swapped this from a `if neq, foo, far` to a `if eq, bar, foo` so that it would match your docstring above.